style.scss

```
@@ -19,49 +36,213 @@
     }
 }
 
+.common-heading2 {
+  position: relative;
+  padding-left: 12px;
+  &:before {
+    position: absolute;
+    content: '';
+    display: block;
+    width: 4px;
+    height: 16px;
+    background-color: #707070;
+    left: 0;
+    top: 50%;
+    transform: translateY(-50%)
+  }
+}
+.common-nav-list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  &__item {
+    list-style: none;
+    background-color: #E9E7DB;
+    border: 1px solid #DCDCDC;
+    width: 100%;
+    height: 100%;
+    margin-left: 8px;
+    margin-right: 8px;
+    border-radius: 50%;
+    box-shadow: 0px 4px 2px 0px rgba(0, 0, 0, .16);
+    flex-shrink: 0;
+    > a {
+        color: #707070;
+        text-decoration: none;
+        width: 100%;
+        height: 100%;
+        display: flex;
+        flex-wrap: wrap;
+        text-align: center;
+        align-content: center;
+      > span {
+        flex-shrink: 0;
+        width: 100%;
+        &:first-child {
+          font-weight: bold;
+        }
+      }
+    }
+  }
+}
+
 .global-nav {
   &__toggle-button {
     display: flex;
-    width: 96px;
+    width: 80px;
     height: 48px;
     border-radius: 8px;
     border: none;
     padding: 8px;
     box-shadow: 1px 4px 8px 1px rgba(0, 0, 0, .8);
     background-color: #fff;
     &:before {
-      content: 'icon';
+      content: '';
       display: block;
-      width: 40px;
+      width: 32px;
       height: 32px;
       flex-shrink: 0;
+      background: url(./img/logo-knicka.png) no-repeat center;
+      background-origin: border-box;
     }
   }
   &__button-line {
-    width: 40px;
+    width: 32px;
     height: 32px;
-    padding-left: 8px;
   }
-  font-size: 2rem;
-  &__list {
-    color: #00F;
+  &__menu {
+    margin: 0;
+    padding: 0;
+    width: 100vw;
+    height: 124px;
+    background-color: rgba(255, 255, 255, .9);
+    box-shadow: 0px 0px 8px 4px rgba(0, 0, 0, .12);
   }
-  &__title {
-  font-weight: bold;
+  &__item {
+    width: 80px;
+    height: 80px;
+    > a {
+      > span {
+        font-size: 1.2rem;
+        &:first-child {
+          font-size: 1.5rem;
+        }
+      }
+    }
   }
 }
 
+.main-content-wrapper {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
 .global-footer {
   font-size: 0.8rem;
   color: #a9a9a9;
   text-align: center;
 }
 
-.statement {
-  font-size: 1.8rem;
+.global-footer-logo {
+  filter: opacity(33%);
+}
```

```suggestion
.global-footer-logo {
  filter: opacity(33%);
}
```
先々、SVGで色指定するように修正する。そのために、SVG画像を用意する。